src/main/kotlin/mate/academy/model/RightTriangle.kt

```diff
@@ -0,0 +1,17 @@
+package mate.academy.model
+
+class RightTriangle(
+    color: Color,
+    private val firstLeg: Double,
+    private val secondLeg: Double
+    ) : Figure (color) {
+
+    override fun getArea(): Double = firstLeg * secondLeg / 2
+
+    override fun draw(): String = ("Figure: ${javaClass.simpleName.lowercase()}, "
+            + "area: ${getArea()} sq. units, "
```

Comment:
`${javaClass.simpleName.lowercase()}` - why do you need this?

Reply:
${javaClass.simpleName.lowercase()} - for reusing universal code in different figures.